modules/ROOT/pages/_partials/showcase-apps/security.adoc

```diff
@@ -0,0 +1,20 @@
+= Security Services
+
+== {device-security-service}
+
+
+From the showcase app:
+
+. Press the *Security* menu item.
+. Press the *Device Trust* sub menu item.
+. View the results of the Device security checks.
+. Navigate to the {grafana-ui}, and view the metrics as described in xref:mobile-metrics.adoc[Analyzing App Usage]
+
+== {app-security-service}
+
```

Comment:
I'd add a line or two here describing what the user will achieve if they execute the steps below.